BenadrylAutomation/src/main/java/com/jnj/pageobjects/ProductPage.java

```diff
@@ -0,0 +1,313 @@
+		String actualHeadingText = ingredientsHeading.getText();
+    	Action.scrollUntilElementVisible(jumpToTitle);
+		Action.performActionwithExtentInfoLog(jumpToHead, "click", "Clicking on : " + jumpToHeadingText);
+        extentInfoLog("Section is displayed : ", Action.isDisplayed(driver, ingredientsSection));
+        Assert.assertEquals(actualHeadingText, sectionHeading);
+        extentInfoLog(jumpToHeadingText + "jump heading verified");
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies the jump to heading.
+     */
+    public void verifyJumpToUsedFor(String jumpToHeading, String sectionHeading) {
+    	this.visitPDP();
+		WebElement jumpToHead= getJumpToHeadings(jumpToHeading);
+		String jumpToHeadingText = jumpToHead.getText();
+		String actualHeadingText = usedForHeading.getText();
+    	Action.scrollUntilElementVisible(jumpToTitle);
+		Action.performActionwithExtentInfoLog(jumpToHead, "click", "Clicking on : " + jumpToHeadingText);
+        extentInfoLog("Section is displayed : ", Action.isDisplayed(driver, usedForSection));
+        Assert.assertEquals(actualHeadingText, sectionHeading);
+        extentInfoLog(jumpToHeadingText + "jump heading verified");
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies the jump to heading.
+     */
+    public void verifyJumpToWarnings(String jumpToHeading, String sectionHeading) {
+    	this.visitPDP();
+		WebElement jumpToHead= getJumpToHeadings(jumpToHeading);
+		String jumpToHeadingText = jumpToHead.getText();
+		String actualHeadingText = warningsHeading.getText();
+    	Action.scrollUntilElementVisible(jumpToTitle);
+		Action.performActionwithExtentInfoLog(jumpToHead, "click", "Clicking on : " + jumpToHeadingText);
+        extentInfoLog("Section is displayed : ", Action.isDisplayed(driver, warningsSection));
+        Assert.assertEquals(actualHeadingText, sectionHeading);
+        extentInfoLog(jumpToHeadingText + "jump heading verified");
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies the jump to heading.
+     */
+    public void verifyJumpToFAQs(String jumpToHeading, String sectionHeading) {
+    	this.visitPDP();
+		WebElement jumpToHead= getJumpToHeadings(jumpToHeading);
+		String jumpToHeadingText = jumpToHead.getText();
+		String actualHeadingText = faqsHeading.getText();
+    	Action.scrollUntilElementVisible(jumpToTitle);
+		Action.performActionwithExtentInfoLog(jumpToHead, "click", "Clicking on : " + jumpToHeadingText);
+        extentInfoLog("Section is displayed : ", Action.isDisplayed(driver, faqsSection));
+        Assert.assertEquals(actualHeadingText, sectionHeading);
+        extentInfoLog(jumpToHeadingText + "jump heading verified");
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies the jump to heading.
+     */
+    public void verifyJumpToReview(String jumpToHeading) {
```

Comment:
@zahinSJI what is this jumpTo??? write a meaningful name for the function. Everywhere you have used jump

Reply:
It is name of a particular set of menu